### DISCUSSION TITLE
CLI compatibility with current projects after upgrade

### DISCUSSION THREAD

**Discussion by celestialvigor** [quasarframework/quasar#13924] [CLI - PWA mode] [answered]
Hi there,

Something that I didn't understand yet from reading the docs and searching here:
Let's say I global install the quasar CLI now and create projects. Will upgrading the CLI version in the future affect its compatibility with current projects? Can I come back and "quasar dev" or "quasar build" from that new CLI version? Does this break between major versions?
Just a quick run down would be appreciated.

**Comment by MilosPaunovic** [comment, marked as answer]
Nope, it doesn't break, `QuasarCLI` supports both versions of Quasar.
